### DISCUSSION TITLE
[Q&A] 当插件只打开了一个的时候，是否可以不再去detect应该用哪一个插件

### DISCUSSION THREAD

**Discussion by ccdanpian** [lobehub/lobehub#900] [General Question] [answered]
### 🥰 需求描述 | Feature Description

当插件只启用了一个的时候，或者用户在输入问题的时候指定了用哪个插件，直接使用这个插件，不再去进行检测。这也也降低了检测失败的概率。

### 🧐 解决方案 | Proposed Solution

直接使用指定插件，不再去进行检测。

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment, marked as answer]
ai的检测还包含了是否在对话中使用该插件。所以这个流程是省略不了的

[a comment by an automated account is not shown]
